selenium/codenvy-selenium-test/pom.xml

```diff
@@ -147,6 +155,11 @@
             <scope>runtime</scope>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.plugin</groupId>
+            <artifactId>che-plugin-github-server</artifactId>
+            <scope>runtime</scope>
```

Comment:
I checked it again and realized that **runtime** scope is really redundant here.
Thank you for notice, Sergii!